src/status_im2/contexts/syncing/find_sync_code/view.cljs

```diff
@@ -0,0 +1,31 @@
+(ns status-im2.contexts.syncing.find-sync-code.view
+  (:require [status-im2.contexts.syncing.syncing-instructions.view :as syncing-instructions]))
+
+(defn view
+  []
+  [syncing-instructions/instructions
```

Comment:
just wondering why the component is needed. is it like really reusable?

Reply:
Yes, It's reusable.
https://github.com/status-im/status-mobile/pull/16364#discussion_r1239571490